packages/manager/modules/pci/src/components/pci-maintenance-banner/template.html

```diff
@@ -24,7 +33,9 @@
             rel="noopener"
             data-translate="pci_projects_maintenance_banner_info_link"
             data-ng-href="{{:: $ctrl.steins[0].travaux }}"
-            data-ng-click="$ctrl.onMaintenanceLinkClick()">
-            ></a>
+            data-ng-click="$ctrl.onMaintenanceLinkClick()"
+        >
+            ></a
+        >
```

Comment:
@maximeVaillant can you exclude these changes from the PR?
Looks like the formatting is not proper.

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/components/pci-maintenance-banner/template.html#L27
Ok it's not formatting, code is not proper ;)
I fix it